docs/source/building/linux.rst

```diff
@@ -32,7 +32,7 @@ Install Requirements
           libopus-dev \
           libpulse-dev \
```

Comment:
Can you also add `libminiupnpc-dev` (Debian/Ubuntu) and `miniupnpc-devel` (Fedora) while you're here?

We missed that in #1971